src/gui/sdlmain.cpp

```diff
@@ -2065,6 +2067,9 @@ static void GUI_StartUp(Section * sec) {
 	sdl.mouse.xsensitivity = p3->GetSection()->Get_int("xsens");
 	sdl.mouse.ysensitivity = p3->GetSection()->Get_int("ysens");
 
+	// Apply raw mouse input setting
+	sdl.mouse.raw_input = section->Get_bool("rawmouseinput");
+
```

Comment:
Suggest eliminating the `sdl.mouse.raw_input` struct variable and replace this:

``` c++
sdl.mouse.raw_input = section->Get_bool("rawmouseinput");
```

with:

``` c++
SDL_SetHint(SDL_HINT_MOUSE_RELATIVE_MODE_WARP, section->Get_bool("rawmouseinput") ? "0" : "1");
```

Reply:
@NicknineTheEagle - can you address this?

Reply:
Why, though? All the mouse settings such as sensitivity are stored in that struct, that's why I put raw input there, too.

Reply:
I agree that saving this user setting in a persistent struct makes sense if it needs to be referenced by more than one chunk of code (and/or multiple times) during runtime.

However in this case, we don't have the above conditions, and can conf setting can simply set this SDL switch the moment we parse the value - and be done with it. This also improves code locality at the same time.

Reply:
Fair point.